internal/eva/login.go

```
@@ -34,17 +33,17 @@ func (c *Client) Login(ctx context.Context, credentials LoginCredentials) error
 	}
 
 	if resp.StatusCode() != 200 {
-		tflog.Trace(ctx, "Login failed.", "Status code", resp.StatusCode(), "body", resp.String())
+		tflog.Debug(ctx, "Request failed.", "Status code", resp.StatusCode(), "body", resp.String())
 
 		return errors.New("Login failed.")
 	}
 
-	var loginResponse LoginResponse
-	if err := json.Unmarshal([]byte(resp.Body()), &loginResponse); err != nil {
+	var jsonResp LoginResponse
+	if err := json.Unmarshal([]byte(resp.Body()), &jsonResp); err != nil {
 		panic(err)
```

Better not to panic here but rather bubble the error up